test/images/iperf/Dockerfile

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google_containers/ubuntu-slim:0.6
+FROM BASEIMAGE
+
+CROSS_BUILD_COPY qemu-ARCH-static /usr/bin/
```

Comment:
QEMUARCH instead of ARCH makes it clearer and possible to use ARCH for other purposes...